### DISCUSSION TITLE
Highlighting subscriptions and/or folder to consolidate them in to one folder

### DISCUSSION THREAD

**Discussion by luzpaz** [ViennaRSS/vienna-rss#274] [Ideas]
OSX v10.8.5 
Vienna Version 3.0.0 RC 1 :6ccabc5:
In Mac OSX finder there is an option to highlight 2 or more finder window items (files or folders) and then click on File -> 'New Folder with Selection ($x items)' where x is the number of highlighted items. Then all of the $x highlighted items are put in to a new folder. This would be a lovely feature to include in Vienna with the addition of adding the function in to the right mouse key pop-up menu as well. 

Thanks!

**Comment by Rand72889** [comment]
Well you can select multiple feeds by holding comand and selecting - than simply drag the where or in which folder you want. Isnt that enough ?
